test/integration/builtin/tags/assign.ts

```diff
@@ -69,6 +69,11 @@ describe('tags/assign', function () {
     const html = await liquid.parseAndRender(src)
     return expect(html).to.equal('-6')
   })
+  it('should allow reassignment', async function () {
+    const src = '{% assign var = 1 %}{% assign var = 2 %}{{ var }}'
+    const html = await liquid.parseAndRender(src)
+    return expect(html).to.equal('2')
+  })
```

Comment:
Unrelated to the bug, but I figured it was good to rule this out as the source of the issue.